src/dsmr/reader.h

```diff
@@ -218,10 +221,10 @@ class P1Reader {
     Stream *stream;
     uint8_t req_pin;
     enum class State : uint8_t {
-      DISABLED,
-      WAITING,
-      READING,
-      CHECKSUM,
+      DISABLED_STATE,
+      WAITING_STATE,
+      READING_STATE,
+      CHECKSUM_STATE,
```

Comment:
Arduino for ESP8266 already defines a macro "DISABLED" that conflicts with
this type definition.
Op 29 mei 2016 11:57 schreef "Matthijs Kooijman" <notifications@github.com>:

> In src/dsmr/reader.h
> <https://github.com/matthijskooijman/arduino-dsmr/pull/3#discussion_r65000969>
> :
>
> > @@ -218,10 +221,10 @@ class P1Reader {
> >      Stream *stream;
> >      uint8_t req_pin;
> >      enum class State : uint8_t {
> > -      DISABLED,
> > -      WAITING,
> > -      READING,
> > -      CHECKSUM,
> > +      DISABLED_STATE,
> > +      WAITING_STATE,
> > +      READING_STATE,
> > +      CHECKSUM_STATE,
>
> Why change these?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/matthijskooijman/arduino-dsmr/pull/3/files/27e2ace07f866f0285c58f57b6690c9dfc280bab#r65000969>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB7KB7I0OqSqrLrpNEAaLqLtKZ_wZIK2ks5qGWMhgaJpZM4Im8LH>
> .
>